src/application-base/ApplicationBase.jsx

```diff
@@ -49,9 +50,22 @@ const ApplicationBase = ({
   locale, themeName, children, unloadPromptIsDisabled,
 }) => {
   const registeredPromptsRef = useRef();
+  const [messages, setMessages] = useState();
 
   const { localeOverride } = useTestOverrides(); // Allows us to test deployed applications in different locales.
 
+  const finalLocale = localeOverride || locale || browserLocale;
+
+  useEffect(() => {
+    if (finalLocale !== undefined) {
+      i18nLoader(finalLocale).then(translationsModule => setMessages(translationsModule.default)).catch((error) => {
+        // eslint-disable-next-line no-console
+        console.error(error);
```

Comment:
Use the terra-application logger here:
https://engineering.cerner.com/terra-ui/application/terra-application/utilities/logger